javascript-objects/main.js

```diff
@@ -0,0 +1,56 @@
+const student = {
+  firstName: 'Jack',
+  lastName: 'Almadjian',
+  age: 22,
+};
+
+const fullName = `${student.firstName} ${student.lastName} ${student.age}`;
+
+console.log(`Value of fullName: ${fullName}`);
+
+student.livesInIrvine = false;
+
+console.log(`Value of student.livesInIrvine: ${student.livesInIrvine}`);
+student.previousOccupation = 'Assistant Mechanic';
+
+console.log(student.livesInIrvine);
+
+console.log('Value of student:', student);
+
+console.log(
+  `Value of student.previousOccupation: ${student.previousOccupation}`
+);
+
+// vehicle object creation
+
+const vehicle = {
+  make: 'Bmw',
+  model: '440i',
+  year: 2017,
+};
+//
+vehicle.color = 'black';
+
+vehicle.isConvertible = false;
```

Comment:
This still needs to be addressed!

This should be done using `bracket notation` rather than `dot notation`